src/runtime/exceptions.cs

```diff
@@ -181,6 +197,7 @@ internal static void SetArgsAndCause(IntPtr ob)
 
             if (e.InnerException != null)
             {
+                // Note: For an AggregateException, InnerException is only the first of the InnerExceptions.
```

Comment:
The note is probably not needed, as an instance of `AggregateException` will itself have a property `InnerExceptions`. E.g. in this case `ob.InnerExceptions`.

Reply:
I think the note is needed because this behaviour was surprising to me (and I consider it a bug in Python.NET that it defines the cause to be the first InnerException only).